SchedulerBot/SchedulerBot.Database.Repositories/Repositories/ServiceUrlRepository.cs

```diff
@@ -10,16 +9,19 @@ namespace SchedulerBot.Database.Repositories
 	/// <inheritdoc cref="IServiceUrlRepository"/>
 	public class ServiceUrlRepository : BaseRepository<ServiceUrl>, IServiceUrlRepository
 	{
-		public ServiceUrlRepository(DbContext dbContext)
+		/// <summary>
+		/// Initializes a new instance of the <see cref="ServiceUrlRepository"/> class.
+		/// </summary>
+		/// <param name="dbContext">The database context.</param>
+		public ServiceUrlRepository(SchedulerBotContext dbContext)
 			: base(dbContext)
 		{
 		}
 
 		/// <inheritdoc/>
 		public async Task<ServiceUrl> GetByAddressAsync(string address)
 		{
-			ServiceUrl serviceUrl =
-				await DbSet.FirstOrDefaultAsync(url => url.Address.Equals(address, StringComparison.OrdinalIgnoreCase));
+			ServiceUrl serviceUrl = await DbSet.FirstOrDefaultAsync(url => url.Address == address);
```

Comment:
In case we cannot use `Equals` method. It would be great to compare strings in lower or upper case